docs/source/managed-federation/federated-schema-checks.mdx

```diff
@@ -2,44 +2,33 @@
 title: Federated schema checks
 ---
 
-> To learn about schema checks for non-federated graphs, see [Schema checks](https://www.apollographql.com/docs/studio/schema-checks/).
+> For an introduction to schema checks, see [Schema checks](https://www.apollographql.com/docs/studio/schema-checks/).
 
-Federation enables teams to work independently on implementing services without needing to coordinate on a monolithic schema. However, this increase in autonomy requires controls to ensure that teams respect [defined dependencies](../federation-spec/) ensure that graph composition succeeds.
+Federation enables teams to work independently on subgraphs without needing to coordinate on a monolithic schema. However, this increase in autonomy requires controls to ensure that teams respect [defined dependencies](../federation-spec/) ensure that supergraph composition succeeds.
 
-In particular, along with validating overall schema changes against known operations, running `apollo service:check` for a federated service will ensure that the overall graph still composes to a valid schema, and will output any violated dependencies if present.
+In particular, along with validating overall schema changes against known operations, running `rover subgraph check` for a subgraph helps ensure that all subgraph schemas still compose to a valid supergraph schema, and will output any violated dependencies if present.
 
-With a federated graph, use the `apollo service:check` command to validate individual service changes by adding the `--serviceName` flag.
-
-When you run `apollo service:check` on a federated service, Apollo Studio runs composition on the proposed capabilities with the current list of federated services and their capabilities, making sure that the composition is successful. That composed schema is then compared against the most recently registered schema to confirm that those changes are safe. If composition fails, then validation ends and results are returned to the user. Note that running `apollo service:check` never updates the graph.
+When you run `rover subgraph check` on a particular subgraph, Apollo Studio runs composition on the proposed capabilities with the current list of subgraphs and their capabilities, making sure that the composition is successful. That composed schema is then compared against the most recently registered schema to confirm that those changes are safe. If composition fails, then validation ends and results are returned to the user. Note that running `rover subgraph check` never updates the graph.
```

Comment:
```suggestion
When you run `rover subgraph check` on a particular subgraph, Apollo Studio attempts to compose a supergraph using the schemas from all of the other published subgraphs and the new proposed subgraph schema. If composition succeeds, the composed schema is compared against the most recently registered schema to confirm that no known clients are impacted by the proposed changes. If composition fails, then validation ends and results are returned to the user. Note that running `rover subgraph check` never updates the graph.
```

I took a shot at this one, though I still don't love my wording 😆 `rover subgraph check` remains one of the most complicated and hard to understand commands. Would love to workshop this one some more.

I propose removing mentions of "capabilities" here since it feels like unnecessary jargon to me (it's actually the first I'm hearing of the term in relation to GraphQL actually). Maybe that would be a mistake though 🤷